core/templates/pages/admin-page/activities-tab/admin-dev-mode-activities-tab.component.html

```diff
@@ -77,13 +77,14 @@ <h3>Reload a single collection</h3>
 
   <mat-card class="oppia-page-card oppia-long-text oppia-long-text-card">
     <div class="container ml-0">
-      <h3>Load dummy new structures data (only admins)</h3>
+      <h3>Load dummy new structures data (only curriculum admins)</h3>
       <div class="row">
         <span class="col-lg-8 col-md-8 col-sm-8">
-          Loads 2 topics (one of which is empty), a story and three skills (two in a subtopic)
-          attached to the first topic and 3 questions in the first skill and 1 question in the second and third skills, respectively. 3 explorations
-          are loaded and associated with the story. Translation and skill question opportunties
-          are populated on the Contributor Dashboard.
+          Loads a published topics , a story and three skills (two in a subtopic)
```

Comment:
topics --> topic

Remove space before the comma.

Reply:
It is done